### PR TITLE
fix(InputObserver): fix destPos for moveDistance

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#127

### CODE REVIEW

File: test/unit/InputObserver.spec.js [hunk 1]
```diff
@@ -105,6 +105,42 @@ describe("InputObserver", function () {
         this.inst.change(inputType, {}, { y: sign * 40 });
         this.inst.release(inputType);
       });
+
+      it(`should check delta that dragged bounce area (direction: ${direction})`, done => {
```
Comment (jongmoon) [inline test/unit/InputObserver.spec.js]: @daybrush What is the difference with above test `should check delta that dragged out of bounce area(direction: ${direction})`?

Comment (daybrush) [reply test/unit/InputObserver.spec.js]: @jongmoon out of bounce area is really outside except bounce.

File: src/InputObserver.ts [hunk 2]
```diff
@@ -80,15 +80,15 @@ export class InputObserver implements IInputTypeObserver {
...
 		// for outside logic
 		destPos = map(depaPos, (v, k) => v + (offset[k] || 0));
+		this.moveDistance && (this.moveDistance = destPos);
 		// from outside to inside
 		if (this.isOutside &&
 			this.axm.every(depaPos, (v, opt) => !isOutside(v, opt.range))) {
 			this.isOutside = false;
 		}
 		destPos = this.atOutside(destPos);
-		this.moveDistance && (this.moveDistance = destPos);
```
Comment (jongmoon) [inline src/InputObserver.ts]: `depaPos = this.atOutside(depaPos);`

File: test/unit/InputObserver.spec.js [hunk 3]
```diff
@@ -105,6 +105,46 @@ describe("InputObserver", function () {
...
+          done();
+        });
+
+        // When
+        const inputType = {
+          axes: ["x"]
+        };
+        const sign = direction > 0 ? 1 : -1;
+        this.inst.hold(inputType);
+				// Move them approximately 150 by slope bounce to reach the end.
```
Comment (jongmoon) [inline test/unit/InputObserver.spec.js]: @daybrush check indentation

File: test/unit/InputObserver.spec.js [hunk 4]
```diff
@@ -105,6 +105,46 @@ describe("InputObserver", function () {
...
+
+        this.axes.setTo({ x: depaPos }, 0);
+
+        let isFirstTime = false;
+        this.axes.on("change", ({ delta }) => {
+          // Then
+          if (!isFirstTime) {
+            // bounce area
+            isFirstTime = true;
+            return;
```
Comment (WoodNeck) [inline test/unit/InputObserver.spec.js]: shouldn't this have to be `isFistTime = true` then `false` as name suggests?

**Pull request merged**